app/src/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -337,18 +339,33 @@ protected void addQuestionText(final FormEntryPrompt p) {
         String audioURI = p.getAudioText();
         String videoURI = p.getSpecialFormQuestionText("video");
         String qrCodeContent = p.getSpecialFormQuestionText("qrcode");
+        String markdownText = p.getMarkdownText();
+
 
         // shown when image is clicked
         String bigImageURI = p.getSpecialFormQuestionText("big-image");
 
         // Add the text view. Textview always exists, regardless of whether there's text.
         mQuestionText = new TextView(getContext());
-        mQuestionText.setText(p.getLongText());
-        mQuestionText.setTextSize(TypedValue.COMPLEX_UNIT_DIP, mQuestionFontsize);
-        mQuestionText.setTypeface(null, Typeface.BOLD);
-        mQuestionText.setPadding(0, 0, 0, 7);
+
         mQuestionText.setId(38475483); // assign random id
 
+        // if we have markdown, use that.
+        if(markdownText != null){
+            mQuestionText.setText(stylize(markdownText));
```

Comment:
I don't think this does depend on the flag? It'll use the markdown if its present in the form